src/views-components/dialog-create/dialog-collection-create.tsx

```diff
@@ -3,113 +3,38 @@
-                ) > 0;
-                return (
-                    <Dialog
-                        open={open}
-                        onClose={handleClose}
-                        fullWidth={true}
-                        maxWidth='sm'
-                        disableBackdropClick={true}
-                        disableEscapeKeyDown={true}>
-                        <form onSubmit={handleSubmit((data: any) => onSubmit(data, files))}>
-                            <DialogTitle id="form-dialog-title">Create a collection</DialogTitle>
-                            <DialogContent className={classes.formContainer}>
-                                <Field name="name"
-                                    disabled={submitting}
-                                    component={TextField}
-                                    validate={COLLECTION_NAME_VALIDATION}
-                                    label="Collection Name" />
-                                <Field name="description"
-                                    disabled={submitting}
-                                    component={TextField}
-                                    validate={COLLECTION_DESCRIPTION_VALIDATION}
-                                    label="Description - optional" />
-                                <FileUpload
-                                    files={files}
-                                    disabled={busy}
-                                    onDrop={files => this.props.dispatch(collectionUploaderActions.SET_UPLOAD_FILES(files))} />
-                            </DialogContent>
-                            <DialogActions className={classes.dialogActions}>
-                                <Button onClick={handleClose} className={classes.button} color="primary"
-                                    disabled={busy}>CANCEL</Button>
-                                <Button type="submit"
-                                    className={classes.lastButton}
-                                    color="primary"
-                                    disabled={invalid || busy || pristine}
-                                    variant="contained">
-                                    CREATE A COLLECTION
-                            </Button>
-                                {busy && <CircularProgress size={20} className={classes.createProgress} />}
-                            </DialogActions>
-                        </form>
-                    </Dialog>
-                );
-            }
-        }
-    );
+import { FormDialog } from '~/components/form-dialog/form-dialog';
+import { CollectionNameField, CollectionDescriptionField } from '~/views-components/form-fields/collection-form-fields';
+import { FileUpload } from '~/components/file-upload/file-upload';
+
+// interface DialogCollectionDataProps {
+//     open: boolean;
+//     handleSubmit: any;
+//     submitting: boolean;
+//     invalid: boolean;
+//     pristine: boolean;
+//     files: UploadFile[];
+// }
```

Comment:
Why not removed?

Reply:
Because, we created a task about improve files for collection. I left for this.